database/passwordIgnore.js

```diff
@@ -0,0 +1,3 @@
+const PASSWORD = 'Wantagh1!@#';
```

Comment:
Might want to change your password... and delete this file from your repo. It will always be here in your git history though...

Reply:
Thanks for pointing this out to me. I put this file in my git ignore so I don't know why it was added.